app/services/adjusted_fees/create_service.rb

```diff
@@ -4,16 +4,17 @@ module AdjustedFees
   class CreateService < BaseService
     Result = BaseResult[:fee, :adjusted_fee]
 
-    def initialize(invoice:, params:)
+    def initialize(invoice:, params:, preview: false)
```

Comment:
Since we have more and more of these "previous", "draft"... flags, could you add a small comment to explain what the impact of passing this flag for future reader :)